Notebooks/historical_vote_proportions.ipynb

```diff
@@ -0,0 +1,1205 @@
+    "from google.auth.credentials import AnonymousCredentials\n",
+    "from cdp_backend.database import models as db_models\n",
+    "\n",
+    "\n",
+    "## Libraries for working with the data files\n",
+    "import pandas as pd\n",
+    "\n",
+    "## For plotting\n",
+    "from matplotlib import pyplot as plt\n",
+    "import numpy as np"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "***\n",
+    "\n",
+    "<center> <h2>Connect to the data base</h2> </center>\n",
+    "\n",
+    "***"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## To start off we will connect to Seattle's data base.\n",
+    "seattle = \"cdp-seattle-staging-dbengvtn\"\n",
+    "fireo.connection(client=Client(\n",
+    "    project=seattle,\n",
+    "    credentials=AnonymousCredentials()\n",
+    "))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "***\n",
+    "\n",
+    "<center> <h3> Fetch all historical votes from data base</h3> </center>\n",
+    "\n",
+    "***"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[<cdp_backend.database.models.Vote at 0x7f904a56db10>,\n",
+       " <cdp_backend.database.models.Vote at 0x7f904a56dbd0>,\n",
+       " <cdp_backend.database.models.Vote at 0x7f904aa98c50>,\n",
+       " <cdp_backend.database.models.Vote at 0x7f904aa98e90>,\n",
+       " <cdp_backend.database.models.Vote at 0x7f904aa98f90>]"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "############ Select all of the records from the Vote table ############\n",
+    "\n",
+    "## If you want to specify a certain amount of records, just use an argument within fetch() function.\n",
+    "seattle_voting_records = list(db_models.Vote.collection.fetch()) \n",
+    "\n",
+    "## Preview file\n",
+    "seattle_voting_records[0:5]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Total Voting Records To Date:\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "3368"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "## Function to calculate total voting records to date:\n",
+    "\n",
+    "def total_vote_records(list_of_objects):\n",
+    "    '''\n",
+    "    Takes a list of database objects and prints out the length.\n",
+    "    '''\n",
+    "    total_records = len(list_of_objects)\n",
+    "    print(\"Total Voting Records To Date:\")\n",
+    "    return(total_records)\n",
+    "\n",
+    "total_vote_records(seattle_voting_records)"
+   ]
```

Comment:
Dont know how much value this cell is adding. Why not just do:

```python
print(len(seattle_voting_records))
```

Just seems a bit basic.

Reply:
I was using this in the  plotting function before and went another way and forgot to change it...
Done